src/Tests/dotnet-new.Tests/DotnetClassTemplateTests.cs

```diff
@@ -121,6 +121,10 @@ await engine.Execute(options)
         [InlineData("interface", "11.0", "net7.0")]
         [InlineData("interface", "10.0", "net6.0")]
         [InlineData("interface", "9", "netstandard2.0")]
+        [InlineData("module")]
+        [InlineData("module", "11.0", "net7.0")]
+        [InlineData("module", "10.0", "net6.0")]
+        [InlineData("module", "9", "netstandard2.0")]
         public async void DotnetVisualBasicClassTemplatesTest(
             string templateShortName,
             string langVersion = "",
```

Comment:
not related to PR, but it would be good to add to those item tests: let's build the project once the item template is added, and see if it's can be built.

Reply:
done